config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += %w( discovery/show.css )
```

Comment:
please use the plugin interface to handle this.. see https://github.com/theforeman/foreman_openscap/blob/94eea5947265ed270b0176f6f94ca80d5923dab4/lib/foreman_openscap/engine.rb#L31

Reply:
removed it from at the moment, will do it properly on the next pr we talked about.